src/System.Unsafe/pkg/System.Unsafe.builds

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Unsafe.pkgproj" Condition="'$(OS)'=='Windows_NT'" />
```

Comment:
Should we have an issue for the Windows_NT only issue and include the issue # here in a comment?